components/PodLogin.jsx

```diff
@@ -0,0 +1,77 @@
+import { Logo } from './logo';
+import { Formik } from 'formik'
+import { ArrowRight as ArrowRightIcon } from './icons'
+import { Input } from './inputs';
+import { ArrowCircleRight } from './icons'
+import Image from 'next/image';
+import forestImage from '../public/forest-landscape.png';
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ')
+}
+
+export default function PodLogin({ avatarImgSrc }) {
+  return (
+    <div className="flex flex-row items-stretch relative h-screen overflow-hidden">
+      <div className="relative z-10 bg-gradient-to-br from-my-green via-ocean to-my-purple p-14 text-white text-sm h-full md:w-1/2 w-full">
+        <div className="relative w-4/5">
+          <Logo className='z-0 absolute -ml-44 -mt-16 transform scale-105 opacity-5' />
+        </div>
+        <div className="text-2xl md:mt-36">
+         Please login to your
+        </div>
+        <div class="md:text-8xl text-4xl font-black">solid pod</div>
+      </div>
+      <div className="flex flex-row z-10 w-full mt-36 md:mt-0 md:w-1/2 absolute md:relative">
+        <div className="flex flex-col md:mt-36 px-20 flex-grow">
+          <Formik>
+            <div className="text-white">
+              Username
+              <Input type="text" name="username"
+                className="flex flex-row mt-2" />
+            </div>
+          </Formik>
+          <Formik>
+            <div className="text-white mt-2">
+            Password
+            <Input type="text" name="password"
+              className="flex flex-row mt-2" />
+            </div>
+          </Formik>
+          <div className="flex flex-col md:flex-row">
+            <button
+              type="button"
+              className={`flex btn-md btn-filled btn-square bg-white text-my-green mt-8 h-10`}
+            >
+              Login <ArrowCircleRight className="w-10 h-10 -m-1 ml-2" />
+            </button>
+            <div className="text-right text-sm text-my-yellow flex-grow mt-3 md:mt-8">
+              Forgot password?
+            </div>
+          </div>
+          <button
+            type="button"
+            className={`text-white text-sm border-t border-b border-opacity-100 border-white rounded-r-sm bg-opacity-0 items-center mt-5 md:mt-8 p-3 md:p-6`}
+          >
+            Login with certificate (WebId-TLS)
```

Comment:
I didn't see "WebId-TLS" on `/login`, `/register`, or `/`, so where is this component used? I also think we can remove this button, I don't think we really care about WebId TLS based login support.

Reply:
oh, it's not - only used in the storybook story. might be worth keeping to help us keep the pod host in sync? ie, when we update styles in garden we'll be able to iterate on the HTML/CSS we need to update the Pod host with